views/pages/home.html

```diff
@@ -2,7 +2,8 @@
 <div class="jumbotron">
 	<h1><span class="accented">Unicorn</span> Rampage</h1>
 	<p class="header-caption">The modern day dungeon master helpmate</p>
-	<button class="btn btn-default" type="button">Create</button>
-	<button class="btn btn-default" type="button">Play</button>
+	<a ng-href="/#/create"><button class="btn btn-default" type="button">Create</button></a>
+	<a ng-href="/#/play">
```

Comment:
We might run into issues with /#/ as a prefix with how i set up routing on S3.
For example, it will work if you are clicking around in the app, but if you copy the URL directly and paste it into a new tab, it will probably break. Just a heads up. Check to see if this happens after you merge it. Wont be a hard fix if it does.